### PR TITLE
Update company edit design

### PR DESCRIPTION
**Pull request by Bestem0r**
# Description

Removes bubbles and adds labels.

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

| Before | After |
| -- | -- |
| image | image |

# Testing

- [x] I have thoroughly tested my changes.

Resolves [ABA-1191](https://linear.app/abakus-webkom/issue/ABA-1191/improve-company-edit-design-page)

### CODE REVIEW

File: app/routes/bdb/components/CompanyEditor.tsx [hunk 1]
```diff
@@ -172,153 +172,106 @@ const CompanyEditor = () => {
...
-        {({ handleSubmit }) => (
-          <form onSubmit={handleSubmit}>
-            <Field
-              name="logo"
-              component={ImageUploadField}
-              aspectRatio={20 / 6}
-              img={company && company.logo}
-            />
+        {({ handleSubmit }) => {
+          return (
```
Comment (ivarnakken) [inline app/routes/bdb/components/CompanyEditor.tsx:176]:
```suggestion
        {({ handleSubmit }) => (
```
Implicit return

File: app/routes/bdb/components/CompanyEditor.tsx [hunk 2]
```diff
@@ -172,153 +172,106 @@ const CompanyEditor = () => {
...
+            <Form onSubmit={handleSubmit}>
```
Comment (ivarnakken) [inline app/routes/bdb/components/CompanyEditor.tsx:177]: you already have a `form` wrapping this

**Review by ivarnakken — APPROVED** [review]
Looks good!

File: app/routes/bdb/components/CompanyEditor.tsx (hunk 2 above)
Comment (Bestem0r) [reply app/routes/bdb/components/CompanyEditor.tsx:177]: So the `<Form onSubmit={handleSubmit}` is not needed at all? Looking at other implementations, there is always a `<form/>` or `<Form/>` inside `<TypedLegoForm/>` @ivarnakken

Comment (ivarnakken) [reply app/routes/bdb/components/CompanyEditor.tsx:177]: yes you need a form, but here you have two

Comment (ivarnakken) [reply app/routes/bdb/components/CompanyEditor.tsx:177]: ooops I believe I have read this wrong! sorry! 😅  I now see that there's only 1 hehe

**Pull request merged**